src/components/Icon/IconBase.js

```diff
@@ -10,22 +10,25 @@ const { block, elem } = bem({
     propsToMods: ['context', 'margin']
 });
 
-const adjustSize = size => {
+const adjustSize = (size, keepWidth) => {
     if (!size) {
-        return null;
+        if (!keepWidth) {
+            return null;
+        }
+        return { width: 'auto' };
```

Comment:
Does this extra condition make a difference with returning `null`? (i.e. won't it be 'auto' implicitly / is there a need to have that extra `!keepWidth` check there?)

Reply:
the default css defines it to `1em`, hence the check. probably would be clearer to move that part of the css here, so that it is all in one place.